SoftLayer/managers/ticket.py

```diff
@@ -80,6 +80,9 @@ def update_ticket(self, ticket_id=None, body=None):
         """
         return self.ticket.addUpdate({'entry': body}, id=ticket_id)
 
+    def upload_attachment(self, ticket_id=None, file=None):
```

Comment:
Not sure what folks' opinion is, but it might be nice to have the filename and data specified as separate parameters to this method and build the file object in here. If not, it would be nice to have docs explaining that the parameter needs to be an object with filename and data properties.